src/platform/Beken/DeviceInfoProviderImpl.cpp

```diff
@@ -0,0 +1,364 @@
+/*

```

Comment:
Is it copy-paste of the code from `examples/providers/DeviceInfoProviderImpl.cpp`? If so, could you use that instead of making another copy?

Reply:
Thanks, we will remove it.